hphp/test/tools/import_zend_test.py

```diff
@@ -1366,6 +1366,10 @@ def parse_headers(zend):
         test = test.replace('EmptyFile.txt', 'md5_EmptyFile.txt')
     if 'ext/standard/tests/strings/lcfirst.php' in full_dest_filename:
         test = test.replace('dummy.txt', 'dummy-lcfirst.txt')
+    if 'ext/standard/tests/strings/strncmp_variation6.php' in full_dest_filename:
+        test = re.sub(r'^var_dump\( strncmp\(\$str1, \$str2, 8\) \)', 'var_dump( strncmp($str1, $str2, 8) > 0)', test)
+        exp  = re.sub(r'range, given in binary format --\nint\(1\)', 'range, given in binary format --\nbool(true)', exp)
```

Comment:
In general we try to keep lines under 80 chars in all files.  All of the style guidelines are [here](https://github.com/facebook/hhvm/blob/master/hphp/doc/coding-conventions.md), and mostly aimed towards C++ development.  Not that this is actually in that guide, but our linter also yells about the 2 spaces before the assignment operator.